sql/core/src/test/scala/org/apache/spark/sql/StatisticsCollectionTestBase.scala

```diff
@@ -374,4 +376,14 @@ abstract class StatisticsCollectionTestBase extends QueryTest with SQLTestUtils
       assert(relation.stats.attributeStats.isEmpty)
     }
   }
+
+  def checkNumBroadcastHashJoins(df: DataFrame, expectedNumBhj: Int, clue: String): Unit = {
+    val plan = EnsureRequirements(spark.sessionState.conf).apply(df.queryExecution.sparkPlan)
```

Comment:
You don't need `EnsureRequirements` to count the number of joins.

Reply:
Thanks! I am correcting this in my next commit.